.github/workflows/workflow.yml

```diff
@@ -110,48 +46,104 @@ jobs:
         with:
           version: v1.28
           working-directory: packages/fetchai/connections/p2p_libp2p/
+
+  common_checks_2:
+    continue-on-error: False
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@master
+        with:
+          python-version: 3.6
```

Comment:
why Python `3.6`? And not e.g. `3.8`?

Reply:
I think for the common check it makes sense to take the lowest version?

Reply:
ok